src/test/scala/no/ndla/searchapi/service/search/SearchConverterServiceTest.scala

```diff
@@ -231,14 +231,14 @@ class SearchConverterServiceTest extends UnitSuite with TestEnvironment {
                                                  TestData.taxonomyTestBundle,
                                                  TestData.emptyGrepBundle)
 
-    searchable1.contexts.size should be(2)
+    searchable1.contexts.size should be(5)
     searchable1.contexts.head.subject.languageValues.map(_.value) should be(Seq("Matte"))
-    searchable1.contexts(1).subject.languageValues.map(_.value) should be(Seq("Historie"))
+    searchable1.contexts(4).subject.languageValues.map(_.value) should be(Seq("Historie"))
 
     searchable4.contexts.size should be(1)
     searchable4.contexts.head.subject.languageValues.map(_.value) should be(Seq("Matte"))
 
-    searchable5.contexts.size should be(2)
+    searchable5.contexts.size should be(3)
     searchable5.contexts.head.subject.languageValues.map(_.value) should be(Seq("Matte"))
     searchable5.contexts(1).subject.languageValues.map(_.value) should be(Seq("Historie"))
```

Comment:
Kan det være en ide å legge inn forventede resultater for alle når sizen utvides?

Reply:
Det er berre en kontekst med to filter som blir to kontekster med ett filter. Men kan alltids legge inn ekstra validering.